hash/ibm5170.xml

```diff
@@ -10355,6 +10355,29 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="doombbs" cloneof="doom">
+		<description>DOOM (1.2, Shareware for BBS SysOps)</description>
+		<year>1994</year>
+		<publisher>Sybex</publisher>
+		<info name="developer" value="id Software" />
+		<info name="version" value="1.2" />
+		<!-- Origin: archive.org -->
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_number" value="0863-1"/>
+			<feature name="part_id" value="Disk 1 of 2"/>
+			<dataarea name="flop" size="1474050">
+				<rom name="Doom_BBS_1.img" size="1474560" crc="e9b64e07" sha1="2f2a7ddc19b1c68696deaa06296684a6b0f4c538"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_number" value="0863-2"/>
+			<feature name="part_id" value="Disk 2 of 2"/>
+			<dataarea name="flop" size="816058">
+				<rom name="Doom_BBS_2.img" size="1262592" crc="8df21831" sha1="50c37cf3b36f4ddfeb315496a4e288d3e64a710a"/>
```

Comment:
1) dataarea size doesn't match rom size
2) File size should be 1474560 for a 1.44Mb disk. In other cases I have padded with 0xF6 which is the default for unused sectors on PC but it doesn't really matter what byte is used.

Reply:
Thanks! PR updated, and I've uploaded the padded image at https://drive.google.com/drive/folders/1thWkdG0ZEtyW19GI7XLpm_w98J5kXtGY?usp=sharing